pkgs/os-specific/gnu/default.nix

```diff
@@ -12,7 +12,7 @@ let
   callPackage = newScope gnu;
 
   gnu = {
-    hurdCross = forceNativeDrv (callPackage ./hurd {
+    hurdCross = forcedNativePackages.callPackage ./hurd {
```

Comment:
Now it doesn't affect those overriden arguments. Honestly I think this is basically dead code with guix existing, I just wanted to do *something* to replace the `forceNativeDrv`. `forcedNativePackages` is also kind of a hack and shouldn't be needed long-term---just use `buildPackages`.